app/com/arpnetworking/metrics/portal/scheduling/JobExecutorActor.java

```diff
@@ -413,8 +413,15 @@ private void jobCompleted(final JobCompleted<?> message) {
                     .addData("scheduled", message.getScheduled())
                     .addData("jobError", message.getError())
                     .log();
-            killSelf();
-            return;
+
+            // Rethrow to crash the actor.
+            //
+            // This will indirectly trigger a reload anyway (see JobExecutorActor#preStart).
+            //
+            // We do this instead of reload directly because our supervisor will rate limit
+            // the resurrection of our actor, whereas reloading directly could lead to
+            // a tight retry loop if the DB issue is not transient.
+            throw e;
```

Comment:
Log-and-throw is usually an antipattern but I don't want to just act like nothing happened (because of unbounded retry), nor do I want to remove the log which provides context about which job is failing.